sdk/samples/iot/pnp/pnp_device_info_component.c

```diff
@@ -1,16 +1,29 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+/*
+ * The Device Info component is a sample that sends (simulated) device information.
+ * It implements the model defined in the DTDLv2
+ * https://devicemodels.azure.com/dtmi/azure/devicemanagement/deviceinformation-1.json.
```

Comment:
Can the README be referenced here instead of adding the url?

Reply:
fixed.